### DISCUSSION TITLE
Overlays Meeting (2024-11-19)

### DISCUSSION THREAD

**Discussion by ralfhandl** [OAI/Overlay-Specification#96] [Meeting Notes]
Zoom: https://zoom.us/j/97548445391 

Schema development
* #95 
* #97
* #71 
* #94

**Comment by lornajane** [comment]
Documentation draft, in case anyone wants to follow along (and keep me accountable!) https://github.com/OAI/learn.openapis.org/pull/115

**Comment by lornajane** [comment]
Git branching restructure

**Comment by lornajane** [comment]
Documentation and runtime instructions for the schema tests.

Where should we put examples of working implementations? Like here's an openapi file, here's an overlay file, here's what you should get as an output of putting those two things together. I have a collection that I'm happy to share, but where?
